packages/core/content-manager/server/services/entity-manager.js

```diff
@@ -40,15 +40,11 @@ const findCreatorRoles = entity => {
 };
 
 // TODO: define when we use this one vs basic populate
-const getDeepPopulate = (uid, populate, depth = 0) => {
+const getDeepPopulate = (uid, populate) => {
```

Comment:
I feel like we can still have circular dependencies 🤔

If a compo has a relation targeting a model and that model has this component. that could create an infinite loop I think

Reply:
I'm not sure! because when it's a relation, the populate is set to `true`, so I don't think it populate the components below. I will try to be sure

Reply:
It seems alright :)

![Capture d’écran 2022-05-16 à 18 44 36](https://user-images.githubusercontent.com/11708220/168642560-3b8c83e9-058f-4f70-9a8a-5e3df89602f7.png)

Reply:
Ok great